src/main/java/org/symphonyoss/symphony/messageml/bi/BiFields.java

```diff
@@ -24,52 +24,52 @@
   HEADER("Headers", BiEventType.MESSAGEML_MESSAGE_SENT),
   IMAGE("Images", BiEventType.MESSAGEML_MESSAGE_SENT),
-  IMAGE_DATA("ImagesData", BiEventType.MESSAGEML_MESSAGE_SENT),
-  IMAGE_URL("ImagesUrl", BiEventType.MESSAGEML_MESSAGE_SENT),
-  LINE_BREAK("LineBreaks", BiEventType.MESSAGEML_MESSAGE_SENT),
+  IMAGE_DATA("Images_Data", BiEventType.MESSAGEML_MESSAGE_SENT),
+  IMAGE_URL("Images_Url", BiEventType.MESSAGEML_MESSAGE_SENT),
+  LINE_BREAK("Line_Breaks", BiEventType.MESSAGEML_MESSAGE_SENT),
   INPUT_STEP("Input_Step", BiEventType.MESSAGEML_ELEMENT_SENT),
   TITLE("Title", BiEventType.MESSAGEML_ELEMENT_SENT),
   DEFAULT("Default", BiEventType.MESSAGEML_ELEMENT_SENT),
   PLACEHOLDER("Placeholder", BiEventType.MESSAGEML_ELEMENT_SENT),
-  STYLE_COLOR("StyleColor", BiEventType.MESSAGEML_ELEMENT_SENT),
-  STYLES_CUSTOM("StylesCustom", BiEventType.MESSAGEML_MESSAGE_SENT),
-  STYLES_CLASS_TEMPO("StylesClassTempo", BiEventType.MESSAGEML_MESSAGE_SENT),
-  STYLES_CLASS_OTHER("StylesClassOther", BiEventType.MESSAGEML_MESSAGE_SENT),
+  STYLE_COLOR("Style_Color", BiEventType.MESSAGEML_ELEMENT_SENT),
+  STYLES_CUSTOM("Styles_Custom", BiEventType.MESSAGEML_MESSAGE_SENT),
+  STYLES_CLASS_TEMPO("Styles_Class_Tempo", BiEventType.MESSAGEML_MESSAGE_SENT),
+  STYLES_CLASS_OTHER("Styles_Class_Other", BiEventType.MESSAGEML_MESSAGE_SENT),
   TYPE("Type", BiEventType.MESSAGEML_ELEMENT_SENT),
   TYPE_MASKED_TRUE("Masked", BiEventType.MESSAGEML_ELEMENT_SENT),
   TYPE_MASKED_FALSE("Normal", BiEventType.MESSAGEML_ELEMENT_SENT),
-  ENTITY_TYPE("EntityType", BiEventType.MESSAGEML_ELEMENT_SENT),
+  ENTITY_TYPE("Entity_Type", BiEventType.MESSAGEML_ELEMENT_SENT),
   LABEL("Label", BiEventType.MESSAGEML_ELEMENT_SENT),
-  OPTIONS_COUNT("OptionsCount", BiEventType.MESSAGEML_ELEMENT_SENT),
+  OPTIONS_COUNT("Options_Count", BiEventType.MESSAGEML_ELEMENT_SENT),
   VALIDATION("Validation", BiEventType.MESSAGEML_ELEMENT_SENT),
   VALIDATION_MAX("Validation_Max", BiEventType.MESSAGEML_ELEMENT_SENT),
   VALIDATION_MIN("Validation_Min", BiEventType.MESSAGEML_ELEMENT_SENT),
   VALIDATION_PATTERN("Validation_Pattern", BiEventType.MESSAGEML_ELEMENT_SENT),
   VALIDATION_OPTIONS("Validation_Options", BiEventType.MESSAGEML_ELEMENT_SENT),
-  HIGHLIGHTED_OPTIONS("HighlightedOptions", BiEventType.MESSAGEML_ELEMENT_SENT),
+  HIGHLIGHTED_OPTIONS("Highlighted_Options", BiEventType.MESSAGEML_ELEMENT_SENT),
   VALIDATION_STRICT("Validation_Strict", BiEventType.MESSAGEML_ELEMENT_SENT),
   REQUIRED("Required", BiEventType.MESSAGEML_ELEMENT_SENT),
-  MESSAGE_LENGTH("MessageLength", BiEventType.MESSAGEML_MESSAGE_SENT),
-  ENTITY_JSON_SIZE("EntitiesJSONSize", BiEventType.MESSAGEML_MESSAGE_SENT),
-  FREEMARKER("UseFreeMarker", BiEventType.MESSAGEML_MESSAGE_SENT),
+  MESSAGE_LENGTH("Message_Length", BiEventType.MESSAGEML_MESSAGE_SENT),
+  ENTITY_JSON_SIZE("Entities_JSON_Size", BiEventType.MESSAGEML_MESSAGE_SENT),
+  FREEMARKER("Use_FreeMarker", BiEventType.MESSAGEML_MESSAGE_SENT),
```

Comment:
Use_Free_Marker?

Reply:
Humm, 
![Screen Shot 2021-06-02 at 15 15 42](https://user-images.githubusercontent.com/39826516/120486517-6b340400-c3b5-11eb-994f-ff26f9149e05.png)
? 

`Use_FreeMarker` looks good

Reply:
yes at the end I kept use_freemarker. I just put every property lower case to use snake_case format.